Program.cs

```diff
@@ -1,10 +1,20 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿/*  Check if the paswword is strong or not.
+The password must contain atleast 6 characters.
+The password must contain atleast one number.
+The password must contain atleast one special character.
+The password must contain atleast one uppercase and lowercase letters. */
+
+using System.Text;
+
+Console.WriteLine ("Enter your password: ");
+string password = Console.ReadLine ();
+string specialCharacter = "!@#$%^&*()-+";
+if (password.Length >= 6) {

```

Comment:
Please provide some line gap between a statement and provide some reasonable line gaps which will improve the code readability.